<launch>
    <node pkg="multi_robots_tf" type="static_roomba_tf2_broadcaster" name="static_roomba_tf2_broadcaster">
        <param name="base_link_frame_id" value="base_link" />
        <param name="lidar_frame_id" value="scan" />
        <param name="thetas_frame_id" value="theta_s" />
        <param name="realsense_frame_id" value="realsense" />
        <rosparam command="load" file="$(find multi_robots_tf)/config/static_roomba_tf.yaml" />
    </node>
    <node pkg="multi_robots_tf" type="roomba_tf2_broadcaster" name="roomba_tf2_broadcaster">
        <param name="base_link_frame_id" value="base_link" />
    </node>
</launch>